tools/schemapi/vega_expr.py

```diff
@@ -0,0 +1,964 @@
+        """
+        Normalize the text content of the signature.
+
+        Examples
+        --------
+        The following definition:
+
+            <a name="sequence" href="#sequence">#</a>
+            <b>sequence</b>([<i>start</i>, ]<i>stop</i>[, <i>step</i>])<br/>
+            Returns an array containing an arithmetic sequence of numbers.
+            ...
+
+        Will yield:
+
+            ['sequence', '(', '[', 'start', ']', 'stop', '[', 'step', ']', ')']
+
+        When called with ``exclude_name=True``:
+
+            ['(', '[', 'start', ']', 'stop', '[', 'step', ']', ')']
+        """
+        EXCLUDE: set[str] = {", ", "", self.name} if exclude_name else {", ", ""}
+        for tok in self._signature_tokens():
+            clean = strip_include_tag(tok[RAW]).strip(", -")
+            if clean not in EXCLUDE:
+                yield from VegaExprNode._split_markers(clean)
+
+    @staticmethod
+    def _split_markers(s: str, /) -> Iterator[str]:
+        """
+        When ``s`` ends with one of these markers:
+
+            ")", "]", "...", " |"
+
+        - Split ``s`` into rest, match
+            - using the length of the match to index
+        - Append match to ``end``
+        - Recurse
+        """  # noqa: D400
+        if s.isalnum():
+            yield s
+        else:
+            end: list[str] = []
+            if s.endswith((CLOSE_PAREN, CLOSE_BRACKET)):
+                end.append(s[-1])
+                s = s[:-1]
+            elif s.endswith(ELLIPSIS):
+                end.append(s[-3:])
+                s = s[:-3]
+            elif s.endswith(INLINE_OVERLOAD):
+                end.append(s[-2:])
+                s = s[:-2]
+            if len(s) == 1:
+                yield s
+            elif len(s) > 1:
+                yield from VegaExprNode._split_markers(s)
+            yield from end
+
+    def _doc_tokens(self) -> Sequence[Token]:
+        """Return the slice of `self.children` that contains docstring content."""
+        for idx, item in enumerate(self):
+            if item[TYPE] == SOFTBREAK and self[idx + 1][TYPE] == TEXT:
+                return self[idx + 1 :]
+            else:
+                continue
+        msg = (
+            f"Expected to find a text node marking the start of docstring content.\n"
+            f"Failed for:\n\n{self!r}"
+        )
+        raise NotImplementedError(msg)
+
+    def _doc_post_process(self, rendered: str, /) -> str:
+        """
+        Utilizing properties found during parsing to improve docs.
+
+        Temporarily handling this here.
+        """
+        # NOTE: Avoids adding backticks to parameter names that are also used in a link
+        # - All cases of these are for `unit|units`
+        pre, post = "[^`_]", "[^`]"
+        pattern = (
+            rf"({pre})\*({'|'.join(self.parameter_names(variadic=False))})\*({post})"
+        )
+        highlight_params = re.sub(pattern, r"\g<1>``\g<2>``\g<3>", rendered)
```

Comment:
# FIXME
- Split out to a function that accepts the parameter names
- Rename: backticks do not mean highlighting
	- Converting italics to backticks for a known set of names